app/serializers/export/user_serializer.rb

```diff
@@ -45,6 +46,10 @@ def relayables
       }
     end
 
+    def blocks
+      object.blocks.map(&:person).map(&:diaspora_handle)
```

Comment:
I think `.map(&:person).map(&:diaspora_handle)` should be done in one `map` step, to not need to first create a full list of persons. But you can add `diaspora_handle` to the delegated methods in the block class, so you can just do `.map(&:person_diaspora_handle)`.

Reply:
Today I learned about delegate methods. Thanks.